### PR TITLE
Performance and other fixes

### PR DESCRIPTION
**Pull request by garraldo**
Implements stopScan upon deiniting 
Adjusts preview video view size to fit entire screen. 
Increases code recognition velocity. 
Minor cosmetic fixes to sample project

### CODE REVIEW

File: VisionScanner/Classes/VisionPreviewView.swift [hunk 1]
```diff
@@ -25,14 +25,22 @@ internal class VisionPreviewView : UIView {
     
     var session: AVCaptureSession? {
         get { return videoPreviewLayer.session }
-        set { videoPreviewLayer.session = newValue }
+        set {
+            videoPreviewLayer.videoGravity = .resize
```
Comment (AlexHmelevskiAG) [inline VisionScanner/Classes/VisionPreviewView.swift]: why resize and not fit? What did I miss?

File: VisionScanner/Classes/VisionScannerViewController.swift [hunk 2]
```diff
@@ -90,7 +91,7 @@ public class VisionScannerViewController: UIViewController {
         }
         
         captureSession.beginConfiguration()
-        captureSession.sessionPreset = AVCaptureSession.Preset.medium
+        captureSession.sessionPreset = AVCaptureSession.Preset.photo
```
Comment (AlexHmelevskiAG) [inline VisionScanner/Classes/VisionScannerViewController.swift]: `aptureSession.sessionPreset = `.photo` will this work?

File: VisionScanner/Classes/VisionPreviewView.swift (hunk 1 above)
Comment (AlexHmelevskiAG) [reply VisionScanner/Classes/VisionPreviewView.swift]: BTW did you check if you change the order - first set session and then Gravity? will it work?
Just curious, it looks like It doesnt work

Comment (garraldo) [reply VisionScanner/Classes/VisionPreviewView.swift]: Yes, both .resize and .resizeAspectFill fill the layer's bounds, but in this case we need .resizeAspectFill. I just updated it.

**Pull request merged**